pkg/volume/util/util.go

```diff
@@ -945,6 +945,20 @@ func CheckPersistentVolumeClaimModeBlock(pvc *v1.PersistentVolumeClaim) bool {
 	return utilfeature.DefaultFeatureGate.Enabled(features.BlockVolume) && pvc.Spec.VolumeMode != nil && *pvc.Spec.VolumeMode == v1.PersistentVolumeBlock
 }
 
+// IsWindowsUNCPath checks if path is prefixed with \\
+// This can be used to skip any processing of paths
+// that point to SMB shares, local named pipes and local UNC path
+func IsWindowsUNCPath(goos, path string) bool {
```

Comment:
In the current state of the code, `goos` is already queried by the callers. So I am just passing it down instead of querying again inside the function. `goos` as a parameter also allows for an easy way to unit-test the function to make sure in other OS environments, there are no unexpected results.